addon/styles/_frost-sidebar.scss

```diff
@@ -1,74 +1,69 @@
 .frost-sidebar {
   position: absolute;
-  left: 0px;
+  max-width: 350px;
+  min-height: 100px;
+  max-height: 250px;
```

Comment:
Likely we'll override these at some point, might be worth documenting in the interface